rpcs3/Emu/RSX/VK/VKGSRender.cpp

```diff
@@ -269,6 +269,17 @@ namespace vk
 		}
 		throw EXCEPTION("Unknown front face value: 0x%X", ffv);
 	}
+
+	VkCullModeFlags get_cull_face(u32 cfv)
+	{
+		switch (cfv)
+		{
+		case CELL_GCM_FRONT: return VK_CULL_MODE_FRONT_BIT;
+		case CELL_GCM_BACK: return VK_CULL_MODE_BACK_BIT;
+		case CELL_GCM_FRONT_AND_BACK: return VK_CULL_MODE_FRONT_AND_BACK;
+		}

```

Comment:
Just add a case default and return VK_CULL_MODE_NONE instead of trapping with a throw